plugins/postcss-custom-media/src/transform-at-media.ts

```diff
@@ -0,0 +1,484 @@
+import { tokenizer, stringify, TokenType, NumberType, TokenIdent } from '@csstools/css-tokenizer';
+import type { CSSToken } from '@csstools/css-tokenizer';
```

Comment:
For now I decided not to implement a full parser for `@media`.
This is a lot of work and the patterns we need to detect and mutate are simple.

This does move a lot of complexity into this plugin that would otherwise exist in a parser:
- split media query lists by `,`
- keep track of bracket nesting depth. (`([{(( deeply nested ))}])`)

But it allows us to move a bit more quickly now and revisit this later to optimise.